pkg/disk/utils.go

```diff
@@ -1066,6 +1066,19 @@ func getDiskVolumeOptions(req *csi.CreateVolumeRequest) (*diskVolumeArgs, error)
 		diskVolArgs.ResourceGroupID = ""
 	}
 
+	// volumeSizeAutoAvailable
+	value, ok = volOptions["volumeSizeAutoAvailable"]
+	if !ok {
+		diskVolArgs.VolumeSizeAutoAvailable = false
```

Comment:
Could use "comma ok" idiom:

```suggestion
	if value, ok = volOptions["volumeSizeAutoAvailable"]; !ok {
		diskVolArgs.VolumeSizeAutoAvailable = false
```